js/public/bindings.js

```diff
@@ -41,25 +44,26 @@ const setBinding = (e, values) => {
         delete bindings[oldId[1]][oldId[2]]
     }
     binding.id = `bind-${type}-${idx}`
-
     bindings[type][idx] = {
         name: getVal('name'),
         invert: getVal('invert') === 'true',
-        action: getVal('action'),
+        action: getVal('action')
     }
 }
-
 const addBinding = values => {
     const labelInput = ({ label, elem, onChange, onInput, ...props }) => {
         const el = document.createElement('label')
-        el.innerText = label
+        if (label) el.innerText = label
         const c = document.createElement(elem)
-        Object.keys(props).forEach(k => c[k] = props[k])
+        Object.keys(props).forEach(k => (c[k] = props[k]))
         if (onChange)
-            c.addEventListener('change', onChange)
+            c.addEventListener('change', e => {
```

Comment:
Unnecessary lambda-wrap, upstream passes onChange by reference, no need to stack an arrow as it adds unnecessary cognitive overhead